scripts/js/commands/checkStaleImages.ts

```diff
@@ -15,8 +15,8 @@ import { globby } from "globby";
 
 import { zxMain, isInstalled } from "../lib/zx.js";
 
-// When users link to this folder, they leave off this prefix.
-const PUBLIC_FOLDER_PREFIX = "public/docs/";
+// When writers link to this folder, they leave off this prefix.
+const PUBLIC_FOLDER_PREFIX = "public/{docs,learning}/";
```

Comment:
Oh, this should actually be `public/`. They shouldn't be leaving off `docs/` or `learning/` anymore. It should be `/docs/images` etc